thu-info-community/thu-info-app
# fix: fix parsing foundation bank payment

**What is the purpose of this PR?**

- [x] Fixing a bug

**Is there a related issue?**

#615 

- [x] I understand that *adding a new feature* before *creating a feature request issue* has a high chance of rejection.

**Please describe your changes.**

Add a check to see if there's table after the selected title, it is a valid item if so and vise versa.

**Are there any possible drawbacks or side-effects?**

I have not fully tested it, so it may cause failure when parsing data.

**How to verify the changes?**

Compile the modified version and check if the item was properly rendered, but as I have some problems in configuring my RN so I can't verify it by myself...
